tutorials/transformers-text-classification.ipynb

```diff
@@ -0,0 +1,751 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "accelerator": "GPU",
+    "colab": {
+      "name": "transformers-text-classification.ipynb",
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "kernelspec": {
+      "display_name": "Python 3",
+      "language": "python",
+      "name": "python3"
+    },
+    "language_info": {
+      "codemirror_mode": {
+        "name": "ipython",
+        "version": 3
+      },
+      "file_extension": ".py",
+      "mimetype": "text/x-python",
+      "name": "python",
+      "nbconvert_exporter": "python",
+      "pygments_lexer": "ipython3",
+      "version": "3.8.8"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "HCS-d1T3znj2"
+      },
+      "source": [
+        "<!-- ---\n",
+        "title: Transformers for Text Classification with IMDb Reviews\n",
+        "date: 2021-09-18\n",
+        "downloads: true\n",
+        "sidebar: true\n",
+        "tags:\n",
+        "  - NLP\n",
+        "  - BERT\n",
+        "  - Transformers\n",
+        "  - Text\n",
+        "  - Beginner\n",
+        "--- -->\n",
+        "# Transformers for Text Classification with IMDb Reviews"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rjZMYxFoznj9"
+      },
+      "source": [
+        "In this tutorial we will fine tune a model from the Transformers library for text classification using PyTorch-Ignite. We will be following the [Fine-tuning a pretrained model](https://huggingface.co/transformers/training.html) tutorial for preprocessing text and defining the model, optimizer and dataloaders. Then we are going to use Ignite for:\n",
+        "* Training and evaluating the model\n",
+        "* Computing metrics\n",
+        "* Setting up experiments and monitoring the model\n",
+        "\n",
+        "According to the tutorial, we will use the [IMDb Movie Reviews Dataset](https://ai.stanford.edu/~amaas/data/sentiment/) to classify a review as either positive or negative."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "sovYyC0Zznj-"
+      },
+      "source": [
+        "## Required Dependencies "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "7XHAD9x7znj_"
+      },
+      "source": [
+        "!pip install pytorch-ignite transformers datasets"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "I80XSUXpvk1i"
+      },
+      "source": [
+        "Before we dive in, we will seed everything using [`manual_seed`](https://pytorch.org/ignite/utils.html#ignite.utils.manual_seed)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "enczLgLTznkH"
+      },
+      "source": [
+        "from ignite.utils import manual_seed\n",
```

Comment:
I very much appreciated what was done in the CIFAR10 notebook. Why not try to do in the same ? I mean to have an import section gathering all required imports, a bit more comments from native pytorch sections (dataloader, optimizer,  model) and use `idist` when possible (`device`, and launch).
What do you think ?

Reply:
Since I mentioned at the beginning of the tutorial that we were following :hugs: tutorial for reference, I didn't want to repeat code that is already explained there plus the imports are introduced as they are used.
I can however, not follow that tutorial verbatim and add more explanation so that our tutorial is standalone.

Reply:
I understand your point. No worries about my comment.